docs/source/sensors/Hayabusa2.rst

```diff
@@ -0,0 +1,9 @@
+Hayabusa2 ONC
```

Comment:
Same comment re: acronyms. I won't add again, as they are used throughout (and whatever you decide is great!)